content/en/docs/concepts/architecture/cgroups.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Cgroup V2
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+On Linux, {{< glossary_tooltip text="control groups" term_id="cgroup" >}}
+are used to constrain resources that are allocated to processes.
+
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}} and the
+underlying container runtime need to interface with control groups to enforce
+[resource mangement for pods and
+containers](/docs/concepts/configuration/manage-resources-containers/) and set
+resources such as cpu/memory requests and limits.
+
+There are two versions of cgroups in linux: cgroupv1 and cgroupv2. Cgroupv2 is
+the new generation of the cgroup API.
+
+<!-- body -->
+
+
+## Cgroup version 2 {#cgroup-v2}
+{{< feature-state for_k8s_version="v1.25" state="stable" >}}
+
+Cgroup v2 is the next version of the cgroup Linux API. Cgroup v2 provides a
+unified control system, which provides enhanced resource management
+capabilities.
+
+The new version offers several improvements over cgroup v1, some of these improvements are:
+
+- cleaner and easier to use API with a unified hierarchy
+- safe sub-tree delegation to containers
+- newer features like Pressure Stall Information
+- enhanced accounting and isolation across multiple resources
+  - accounting for network memory
+
+
+Some kubernetes features exclusively rely on on cgroupv2 for enhanced resource
+management and isolation. For example, the
+[MemoryQoS](/blog/2021/11/26/qos-memory-resources/) feature improves memory QoS
+and relies on cgroupv2 primitives. New upcoming resource management
+capabilities in kubelet will depend on cgroupv2 as well.
+
+
+## Using cgroupv2
+
+To use cgroupv2, it is recommended to use a Linux distribution which enables
+cgroupv2 out of the box. Most new modern linux distributions have switched over
+to cgroupv2 by default.
+
+To check if your distribution is using cgroupv2, follow the steps [below](#check-cgroup-version).
+
+To use cgroupv2 the following requirements must be met:
+
+* OS distribution enables cgroupv2
+* Linux Kernel version is >= 5.8
+* Container runtime supports cgroupv2
+  * [containerd](https://containerd.io/) since 1.4
+  * [cri-o](https://cri-o.io/) since 1.20
+* Kubelet and container runtime are configured to use the [systemd cgroup driver](/docs/setup/production-environment/container-runtimes#systemd-cgroup-driver)
+
+### Linux Distribution cgroupv2 support
+
+Many Linux Distributions have already switched over to use cgroupv2 by default, for example:
```

Comment:
```suggestion
For a list of Linux distributions that use cgroup v2, refer to the [cgroup v2 documentation](https://github.com/opencontainers/runc/blob/main/docs/cgroup-v2.md)
```

Reply:
see below

Reply:
Based on your comments, I'm personally ok with keeping the list, but we should still link to the source so that maintainability doesn't rely solely on this page. Maybe after the list items a line that says "This list is derived from the [runc cgroup v2 documentation]\(link)."

Reply:
Sounds good, added!